src/main/resources/io/jenkins/plugins/coverage/CoverageAction/summary.jelly

```diff
@@ -2,7 +2,7 @@
 <j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
     <t:summary icon="graph.png">
         <div><a href="coverage/">${%Coverage Reports}</a></div>
-        <j:set var="result" value="${it.target}" />
+        <j:set var="result" value="${it.threshTarget}" />
```

Comment:
This object exists for `ThreshHold`, but not for `CoverageAction`. Not sure how it works